File: src/main/java/net/minestom/server/entity/metadata/object/ThrownPotionMeta.java
package net.minestom.server.entity.metadata.object;

import net.minestom.server.entity.Entity;
import net.minestom.server.item.Material;
import org.jetbrains.annotations.NotNull;

public class ThrownPotionMeta extends ObjectEntityMeta {

    public ThrownPotionMeta(@NotNull Entity entity) {
        super(entity, Material.AIR);
    }

}

File: src/main/java/net/minestom/server/entity/metadata/item/ThrownPotionMeta.kt
package net.minestom.server.entity.metadata.item

import net.minestom.server.entity.Entity
import net.minestom.server.entity.Metadata
import net.minestom.server.item.Material

class ThrownPotionMeta(entity: Entity, metadata: Metadata) : ItemContainingMeta(entity, metadata, Material.AIR) {
    companion object {
        val OFFSET: Byte = ItemContainingMeta.Companion.MAX_OFFSET
        val MAX_OFFSET = (OFFSET + 0).toByte()
    }
}